flixel/addons/editors/tiled/TiledImageSet.hx

```diff
@@ -0,0 +1,51 @@
+package flixel.addons.editors.tiled;
+
+import haxe.xml.Fast;
+
+/**
+ * Copyright (c) 2013 by Samuel Batista
+ * (original by Matt Tuttle based on Thomas Jahn's. Haxe port by Adrien Fischer)
+ * This content is released under the MIT License.
+ */
+class TiledImageSet implements Dynamic<String>
```

Comment:
Not sure, what does a `TiledImageSet` look like in the xml compared to a `TiledImageSet`?